lib/kafka/producer.rb

```diff
@@ -225,22 +268,6 @@ def deliver_messages
       end
     end
 
-    # Returns the number of messages currently held in the buffer.
```

Comment:
Nah, it's just the diff that's a bit weird – if you look further up in the file you can see these methods are there, still intact.